searchcore/src/vespa/searchcore/proton/server/bucketmovejobv2.cpp

```diff
@@ -53,6 +53,8 @@ blockedDueToClusterState(const std::shared_ptr<IBucketStateCalculator> &calc)
     return !(clusterUp && nodeUp && !nodeInitializing);
 }
 
+constexpr BucketId RECOMPUTE_BUCKETID;
```

Comment:
Above name indicates that the scope is a single bucket.  But the usage indicates that the scope is all buckets.

Reply:
Agree, RECOMPUTE_TOKEN it is.